drivers/espi/Kconfig

```diff
@@ -112,6 +112,12 @@ config ESPI_PERIPHERAL_DEBUG_PORT_80
 	help
 	  Enables debug Port 80 over eSPI peripheral channel.
 
+config ESPI_PERIPHERAL_EC_HOST_CMD
```

Comment:
Is this different than CONFIG_ESPI_PERIPHERAL_HOST_IO?

Reply:
No, this config is to enable the support of 'EC host command framework'. Please https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/ec_host_cmd_periph/Kconfig#L6 for more detail.

Reply:
Done